core/src/test/java/io/kestra/core/repositories/AbstractWorkerInstanceRepositoryTest.java

```diff
@@ -0,0 +1,10 @@
+package io.kestra.core.repositories;
+
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.BeforeEach;
+
+@MicronautTest(transactional = false)
+public abstract class AbstractWorkerInstanceRepositoryTest {
+
```

Comment:
There is no test?

Reply:
As we don't want those tests to be executed with ElasticSearch, they are in `AbstractJdbcWorkerInstanceRepositoryTest`

Reply:
So it's useless, you can remove it